dockerfile

```diff
@@ -0,0 +1,16 @@
+# base image
+FROM node:12.2.0-alpine
+
+# set working directory
+WORKDIR /app
+
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /app/node_modules/.bin:$PATH
```

Comment:
Waar is dit precies voor nodig?

Reply:
$PATH aan de PATH variabele (van de image) toevoegen, zorgt er voor dat het operating system altijd de (juiste) versies van de commando's in de node_modules kan vinden. Meestal is diet niet nodig maar het wordt wel aangeraden om problemen (met complexere dockerfiles en images met andere versies van commando's) te voorkomen.